tox.ini

```diff
@@ -7,7 +7,9 @@ envlist =
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 install_command = pip install --no-deps {opts} {packages}
-deps = -rrequirements.txt
+deps = 
+    -rrequirements.txt
+    py27: ipaddress
```

Comment:
please put all requirements in requirements.in

also don't see why we need ipaddress??